godot-kotlin/godot-library/build.gradle.kts

```diff
@@ -46,4 +48,31 @@ tasks {
     build {
         finalizedBy(publishToMavenLocal)
     }
+
+    // workaround to upload gradle metadata file
+    // https://github.com/bintray/gradle-bintray-plugin/issues/229
+    withType<BintrayUploadTask> {
+        doFirst {
+            publishing.publications.withType<MavenPublication> {
+                buildDir.resolve("publications/$name/module.json").also {
+                    if (it.exists()) {
+                        artifact(object: FileBasedMavenArtifact(it) {
+                            override fun getDefaultExtension() = "module"
+                        })
+                    }
+                }
+            }
+        }
+    }
+}
+
+//TODO: See how to do with mobile platforms
```

Comment:
I think it is as simple as:

```
OS.LINUX -> arrayOf("kotlinMultiplatform", "metadata", "linux", "android64)
```

Meaning, on linux we also publish the `android64` target.

Reply:
I would like to have those build in parrallel :)

Reply:
Ahh right, something that we can optimise later on. But as a start we can just build everything buildable for a particular os.

Reply:
Yep, I agree.
But as for beta we do not want arm platform, it is not imoportant yet